app/templates/web-form.html

```diff
@@ -0,0 +1,207 @@
+{%- extends 'base-' + display_region + '.html' -%}
+
+{%- from 'components/input/_macro.njk' import onsInput -%}
+{%- from 'components/button/_macro.njk' import onsButton -%}
+{%- from 'components/select/_macro.njk' import onsSelect -%}
+{%- from 'components/radios/_macro.njk' import onsRadios -%}
+{%- from 'components/textarea/_macro.njk' import onsTextarea -%}
+{%- from 'components/panel/_macro.njk' import onsPanel -%}
+
+{%- set messages_dict=dict(get_flashed_messages()|groupby('level')) -%}
+{%- set field_messages_dict=dict(messages_dict['ERROR']|groupby('field')) -%}
+
+{%- set form =  {
+    'method': 'POST',
+    'attributes': {
+        'action': url('WebForm:post', display_region=display_region)
+    }
+} -%}
+
+        {%- set country_options =
```

Comment:
not sure why we have country-options when the URL has already got a country specific part to it. Seems like doubling up .

Reply:
The url country specific part is not necessarily country:
/en is England or Wales in English (and NI users using the wrong url as we can't trap them out here)
/cy is England or Wales in Welsh 
/ni is Northern Ireland in English

Without asking for Country we can't tell if they are in England or Wales on a /en url. If they use the wrong site version (census.gov.uk when in NI) Web form has no way of knowing they are in /ni either without asking. 
It makes a difference because there are 4 form versions, and it saves an email going back to ask the question

Reply:
ok thanks for the explanation